west.yml

```diff
@@ -47,12 +47,24 @@ manifest:
     - name: fatfs
       revision: df96914dd989907f3a5de4cb95b116c2f305820d
       path: modules/fs/fatfs
+    - name: hal_cypress
+      revision: master
```

Comment:
Please set the revision for `hal_cypress` and `hal_silabs` to a tag or SHA, to preserve bisectability.

Reply:
ah, good catch, missed those..